contracts/proxies/LinkerRouterProxy.sol

```diff
@@ -15,19 +15,12 @@ contract LinkerRouterProxy is Proxy {
      * @param owner Address of the owner of the proxy
      * @param params The params to be passed to the _setup function of the implementation.
      */
-    constructor(
-        address implementationAddress,
-        address owner,
-        bytes memory params
-    )
-        Proxy(implementationAddress, owner, params) // solhint-disable-next-line no-empty-blocks
-    {}
+    constructor(address implementationAddress, address owner, bytes memory params) Proxy(implementationAddress, owner, params) {}
 
     /**
      * @dev Override for the `contractId` function in Proxy. Returns a unique identifier for this contract.
      * @return bytes32 Identifier for this contract.
      */
-    // solhint-disable-next-line no-empty-blocks
     function contractId() internal pure override returns (bytes32) {
         // bytes32(uint256(keccak256('remote-address-validator')))
         return 0x5d9f4d5e6bb737c289f92f2a319c66ba484357595194acb7c2122e48550eda7d;
```

Comment:
can you switch all hardcoded hashes to the actual expressions? We have a lot of them, and it's not very maintainable (some of them are already wrong)